index.html

```diff
@@ -211,23 +234,39 @@ <h4>Don't Forget These!</h4>
                 </label>
               </div>
               <!-- form check -->
-              <button type="submit" class="btn btn-info mt-2">Let's Travel</button>
+              <div class="btn-group">
+                <button type="button" class="btn btn-success dropdown-toggle mt-2" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                  Action
+                </button>
+                <div class="dropdown-menu">
```

Comment:
Nice creativity and thought process on adding the drop down menu here! Who is your agent? Tony or Tonya? 😆